scripts/ci_check_no_file_changes.sh

```diff
@@ -37,7 +37,7 @@ do
     REGEX="(^\.github\/workflows\/.*python|^tests\/|^superset\/|^setup\.py|^requirements\/.+\.txt|^\.pylintrc)"
     echo "Searching for changes in python files"
   elif [[ ${CHECK} == "frontend" ]]; then
-    REGEX="(^\.github\/workflows\/.*(frontend|e2e)|^superset-frontend\/)"
+    REGEX="(^\.github\/workflows\/.*(bashlib|frontend|e2e)|^superset-frontend\/)"
```

Comment:
Adding changes in `bashlib.sh` to flag frontend changes, as this file controls many aspects of how dependencies are managed on CI.